installation.md

```diff
@@ -0,0 +1,31 @@
+# Installing Leapp tool
+
+## Fedora 26 and 27
+
+	```shell
```

Comment:
According to the render, this does not work. I believe that the space/tab in front of ``` is not allowed. Also, when you use it this way - empty line before and after the block together with the indentation, it is considered automatically as a block.